src/components/NewGalaxyModal.js

```diff
@@ -117,11 +121,11 @@ export const NewGalaxyModal = withModalDrawer({ width: 675 })(class NewGalaxyMod
 
     const renderDefaultCase = () => {
       return h(Fragment, [
-        div({ style: { ...styles.drawerContent, paddingTop: '0.5rem' } }, [
-          div(['Environment consists of an application and cloud compute.']),
+        div(['Environment consists of an application and cloud compute.']),
+        div({ style: { paddingTop: '0.5rem' } }, [
           div({ style: { ...styles.whiteBoxContainer, marginTop: '1rem' } }, [
```

Comment:
I think this marginTop is too much, I tried `0.5rem` which looked a little better.